citation.cff

```diff
@@ -0,0 +1,28 @@
+cff-version: 1.2.0
+message: If you use this software, please cite it as below.
+title: "CHGNet: Pretrained universal neural network potential for charge-informed atomistic modeling"
```

Comment:
The title of CHGNet on NMI has been changed to CHGNet as a pretrained universal neural network potential for charge-informed atomistic modelling